src/test/streams_tests.cpp

```diff
@@ -379,8 +380,8 @@ BOOST_AUTO_TEST_CASE(streams_buffered_file)
     // by the rewind window (relative to our farthest read position, 40).
     BOOST_CHECK(bf.GetPos() <= 30U);
 
-    // We can explicitly close the file, or the destructor will do it.
-    file.fclose();
+    // Explicitly close the file and check that the close succeeds.
```

Comment:
comments isn't useful anymore, the code already states the same

Reply:
Removed.